release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -865,6 +865,30 @@ Updated guidance around xref:../scalability_and_performance/planning-your-enviro
 
 Use the link:https://access.redhat.com/labs/ocplimitscalculator/[{product-title} Limit Calculator] to estimate cluster limits for your environment.
 
+[id="ocp-4-7-optimizing-data-plane-performance-with-intel-devices"]
+==== Optimizing data plane performance with the Intel vRAN Dedicated Accelerator ACC100
+
+{product-title} {product-version} supports the OpenNESS SR-IOV Operator for Wireless FEC Accelerators.
+
+This Operator supports the requirements of vRAN deployments for low power, cost and latency, whilst also delivering the capacity to manage spikes in performance for a range of use cases. One of the most compute-intensive 4G and 5G workloads is RAN layer 1 (L1) forward error correction (FEC), which resolves data transmission errors over unreliable or noisy communication channels.
+
+Delivering high performance FEC is critical to 5G maintaining high performance as it matures and as more users depend on the network. FEC is now supported on the Intel vRAN Dedicated Accelerator ACC100 card with the OpenNESS SR-IOV Operator for Wireless FEC Accelerator.
+
+[NOTE]
+====
+This is built and shipped by Intel and currently supported with {product-title} 4.7.8. If a different {product-title} version is requested please contact Intel through their premier support portal at link:http://premiersupport.intel.com[Intel® Premier Support Access] or at mailto:openness.n3000.operator@intel.com[openness.n3000.operator@intel.com].
```

Comment:
I suggest extending this guidance from the style guide and replace "This is." I am not sure exactly what is shipped by Intel.

> ! there is, there are verb
Avoid at the beginning of an independent clause. In most cases rewrite to remove the phrase. For example, 
 instead of "There are three recovery control data sets in the product" write "The product includes three recovery control data sets".
 
 Also:
 
 > ! please adverb
Do not use in technical information.